framework/PageDashboard/PageDashboardChart.tsx

```diff
@@ -43,21 +85,39 @@ export function PageDashboardChart(props: {
             />
           }
         >
+          <ChartAxis fixLabelOverlap label={xLabel} style={{ axisLabel: { fontSize: 16 } }} />
           <ChartAxis
-            fixLabelOverlap
-            // tickFormat={(date: string) => `${new Date(date).toLocaleDateString()}`}
-            //  tickFormat={(n) => `${Math.round(n)}`}
+            dependentAxis
+            showGrid
+            label={yLabel}
+            tickFormat={
+              onlyIntegerTicks
+                ? //eslint-disable-next-line @typescript-eslint/no-unsafe-return
+                  (t) => (Number.isInteger(t) ? t : '')
```

Comment:
I am still seeing no tick values on the y-axis when `onlyIntegerTicks` is true- maybe try pushing the changes again?

Reply:
I forgot to push the commit... Sorry. It should be there now with missing props you mentioned in your review.